src/test/resources/test-suite-tck.json

```diff
@@ -38967,6 +38997,11 @@
             "format": "date-time"
           },
           "tests": [
+            {
```

Comment:
I know these are not the TCK tests provided by the JSON Schema team, but to make my life simpler I added the two issues here to for testing. If you don't want this I can revert.